manifests/4.2/cluster-logging.v4.2.0.clusterserviceversion.yaml

```diff
@@ -139,6 +139,7 @@ spec:
           - configmaps
           - secrets
           - serviceaccounts
+          - serviceaccounts/finalizers
```

Comment:
@jcantrill @ewolinetz do we need to add this rbac anywhere else?  Is this CSV the one used by customers?